python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -55,7 +66,19 @@ def __init__(
         """
         self._generate_cli_args = generate_cli_args or ["parse", "--quiet"]
 
+    @public
```

Comment:
Given the other names used here, should this be named something like `prepare_if_local`? Or `ensure_prepared`?

Reply:
`ensure_prepared` would make sense. I think making the change in the [PR updating the API](https://github.com/dagster-io/dagster/pull/22534) would make most sense.